eppo_client/assignment_logger.py

```diff
@@ -1,6 +1,10 @@
 from typing import Dict
+from eppo_client.base_model import BaseModel
+from pydantic import ConfigDict
 
 
-class AssignmentLogger:
+class AssignmentLogger(BaseModel):
+    model_config = ConfigDict(arbitrary_types_allowed=True)
+
```

Comment:
Without this change an error is throw:

```
ERROR test/client_test.py - RuntimeError: no validator found for <class 'eppo_client.assignment_logger.AssignmentLogger'>, see `arbitrary_types_allowed` in Config
```

Is this the "minimal class" in python now?

Reply:
I think it's because this is used by `Config` which is a Pydantic model.

Reply:
fixed